### DISCUSSION TITLE
Button Location

### DISCUSSION THREAD

**Discussion by NothingTV** [vidstack/player#898] [Q&A] [answered]
Hello there,

I recently switched from another player to vidstick and I really love it, the only issue I currently have, is the location of added Buttons like seek buttons. They usually appear between the actual player settings and the video, so they are not clickable.
What is the proper way to add them simply to the other buttons like play etc.?
This is my current Player, the seek Buttons appear on the bottom left behind the play button, not clickable:
```
<media-player
    title="{{ $vod->title }}"
    src=""
    poster=""
    thumbnails=""
    aspect-ratio="16/9"
    crossorigin>

    <media-outlet>
        <media-poster alt="'"></media-poster>
    </media-outlet>
    <media-seek-button seconds="+15"></media-seek-button>
    <media-seek-button seconds="-15"></media-seek-button>

    <media-community-skin></media-community-skin>
</media-player>
```

I'm sorry for asking such a question, I tried everything and read nearly every article in the doc about this topic.

Best regards

**Comment by mihar-22** [comment, marked as answer]
Hey NothingTV! At the moment, there is no way to add additional controls but we're looking into making the default UI customizable. For now, you can only completely style and hide existing controls.

**Reply by NothingTV** [reply]
Thank you for your fast reply! :)
I couldn't find a example yet, is there on in the docs that I miss?

**Reply by mihar-22** [reply]
Example of? Adjustying styles and hiding controls is completely via CSS. You can also customize most parts using [CSS variables](https://www.vidstack.io/docs/player/core-concepts/skins#video-css-vars).

**Reply by NothingTV** [reply]
Hi, thanks again for the answer.
I thought of a concrete example how you would do this. So hide all official elements and show own ones with the proper size etc., I'll figure it out, thanks! :)

**Reply by NothingTV** [reply]
Is there maybe a way for a full example HTML/CSS/JS Code for the player on the demo page?

**Reply by mihar-22** [reply]
Complete examples are coming soon, I'm just wrapped up in a big release but right after it I'll be adding some.

**Reply by sidis405** [reply]
Can't wait for this :) It'd be great to have just some sort of example of markup where to plug in control components to mix and match a Skin ;)
